frontend/src/metabase/entities/schemas.js

```diff
@@ -194,24 +194,26 @@ function addTableAvoidingDuplicates(tables, tableId) {
   return tables.includes(tableId) ? tables : [...tables, tableId];
 }
 
-const useGetQuery = ({ id, ...args }) => {
-  const [dbId, schemaName, opts] = parseSchemaId(id);
+const useGetQuery = query => {
```

Comment:
It was not necessary in this PR, but I added `skipToken` handling here for consistency and to avoid surprises in the future.